app/src/main/java/com/example/albumsapp/parser/PhotoColorParser.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.albumsapp.parser
+
+import android.graphics.Color
+import java.util.*
+
+class PhotoColorParser(private val url: String) {
```

Comment:
Imagine we have 100 photos in RecyclerView. To parse each photo url, we need to create a new object of class `PhotoColorParser` to call one method . Can it be somehow improved? May be we can create only one instance of the parser ?